src/test/java/pl/marcin/project/repository/PurchaseRepositoryListBasedTest.java

```diff
@@ -0,0 +1,154 @@
+package pl.marcin.project.repository;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import pl.marcin.project.model.Cup;
+import pl.marcin.project.model.Customer;
+import pl.marcin.project.model.Purchase;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PurchaseRepositoryListBasedTest {
+    PurchaseRepository purchaseRepository = new PurchaseRepositoryListBased();
```

Comment:
can be private! (same to other test classes)

Reply:
ok